<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>93</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Connection</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background-color: darkgrey;
}
.QPushButton {
  padding: 5px;
  background-color: lightgrey;
  cursor: pointer;
  text-align: center;
  border: none;
  border-radius: 5px;
}

.QPushButton:hover {
    border: 1px solid #595750;
}

.QPushButton:pressed {
  background-color: #303030;
  color: #d4d4d4;
}

.QListWidget{
	border-radius: 5px;
	border: 1px solid #595750;
}


QScrollBar:vertical {
	width: 1em;
}

QPlainTextEdit, QLineEdit{
	border-radius: 5px;
	border: 1px solid #595750;
	background-color: lightgrey;
}</string>
  </property>
  <widget class="QPlainTextEdit" name="nicknameTextField">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="sendButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Send</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
